cache/src/convertable.rs

```diff
@@ -93,7 +101,6 @@ async fn convert_to_new_type<
 		.await
 }
 
-#[async_trait]
```

Comment:
how's this still working, since you removed that crate? is there any particular reason for using async trate here instead of the newer actual async in trait feature?

Reply:
It was origanally done with `async_trait` because it was code written before AFIT was possible.